src/server/scripts/Northrend/IcecrownCitadel/boss_valithria_dreamwalker.cpp

```diff
@@ -363,7 +363,10 @@ class boss_valithria_dreamwalker : public CreatureScript
             }
             else if (_instance->GetBossState(DATA_VALITHRIA_DREAMWALKER) == NOT_STARTED)
```

Comment:
Maybe you can add brackets to this too? <3

Reply:
Ready